functional_testing/Openvcloud/ovc_master_hosted/ACL/a_basic_operations/acl_usergroup_test.py

```diff
@@ -170,15 +170,11 @@ def test003_usergroup_delete_user_from_account(self):
         #. try to delete user1 from this account  by himself should be bad request as user is last admin
         #. add user2 to this account by user1 should succeed
         #. delete user1 from this account by user1 should succeed
-        #. delete created account by user1 should be forbidden
```

Comment:
why did u remove it?

Reply:
/cloudapi/accounts/delete api is removed